### DISCUSSION TITLE
DEYE 12kW doesn't work

### DISCUSSION THREAD

**Discussion by bastel-123** [StephanJoubert/home_assistant_solarman#85] [Q&A]
The profile deye_hybrid.yaml doesn't work for the 12kw hybrid model. The presented values are all invalid. How can I work on to find out the right parameters to customize the yaml file?

**Comment by skupi20** [comment]
The same for deye hybrid 3 phase, all values are wrong.

edit:
Success. I have created configuration for hybrid 3 phase deye inverter :) please be patience, soon i will put it here.

[custom parameters for deye hybrid 3 phase inverter](https://github.com/StephanJoubert/home_assistant_solarman/issues/43)

**Comment by GyroGearl00se** [comment]
I had success with
 https://github.com/StephanJoubert/home_assistant_solarman/issues/88#issuecomment-1339941709
